assignments/lambda-classes.js

```diff
@@ -1 +1,139 @@
 // CODE here for your Lambda Classes
+
+
+class Person {
+    constructor(attributes) {
+        this.name = attributes.name;
+        this.age = attributes.age;
+        this.location = attributes.location;
+        this.gender = attributes.gender;
+}
+
+speak() {
+    return `Hello my name is ${this.name} , I am from ${this.location}`
+}
+} 
+
+class Instructor extends Person {
+    constructor(attributes) {
+    super(attributes);
+    this.specialty = attributes.specialty;
+    this.favLanguage = attributes.favLanguage;
+    this.catchPhrase = attributes.catchPhrase;
+}
+demo(subject) {
+   return `Today we are learning about ${subject}`
+    
+ }   
+    
+
+
+grades(student,subject){
+   return `${student.name} receives a perfect score on ${subject}`
+}
+}
+class Student extends Instructor {
+    constructor(attributes){
+    super(attributes);
+    this.previousBackground = attributes.previousBackground;
+    this.className = attributes.className;
+    this.favSubjects = attributes.favSubjects;
+    }
+    listsSubjects(){
+       return this.favSubjects.forEach(function(item) {
+           console.log(item);
+        });
+
+    }
+
+    PRAssignment(){
+        return `${student.name} has submitted a PR for {subject}`
+    }
+    sprintChallenge(){
+        return `${student.name} has begun sprint challenge on {subject} `
+    }
+}
+
+    class Project_Mananger extends Student {
```

Comment:
again, take a look at what you're extending from here. go over the readme instructions again on which subclasses should be the parent here